src/components/parts/team/TeamMember.astro

```diff
@@ -7,13 +7,18 @@ import SystemMembers from "./SystemMembers.astro";
 export interface Props {
 	member: TeamMember;
 }
-const { member } = Astro.props as Props;
+const { member, isLeader } = Astro.props as Props;
 
 // this is the "N/A" text, for anyone wondering
 const na = t("team:n-a");
 ---
 
 <div class="card is-team-member">
+	{ isLeader &&
+		<span class="icon is-large is-relative is-pulled-right pr-2 pt-2">
```

Comment:
I think a tooltip would be a nice addition there ­— the star doesn't convey anything obvious if you missed the fine text in the final paragraph (which is VERY easy to do if you just want to see a list of names):
> [...] On this page, they will be indicated by a star.

Reply:
I tried to add a tooltip, it just didn't work and I didn't bother looking at what piece of the HTML hell is breaking it